geomstats/visualization/stiefel_manifold.py

```diff
@@ -0,0 +1,450 @@
+"""Stiefel manifold module."""
+import matplotlib.pyplot as plt
+import numpy as np
+
+import geomstats.backend as gs
+from geomstats.geometry.hypersphere import Hypersphere
+from geomstats.visualization import Sphere
+
+# %%
+"""
+    Consider <n=3, p=1> Stiefel Manifold.
+    Dimension is 2, which can be represented as sphere.
+"""
+
+
+class StiefelSphere:
+    """
+    Class used to plot something in Stiefel shape space of 2D triangles.
```

Comment:
Add reference showing that the stiefel space St(2, 3) is indeed the shape space of triangles.
Look at other code in geomstats repo to see how to properly add a reference in a docstring